### PR TITLE
Utility

### PR DESCRIPTION
**Pull request by mohmamad**

### CODE REVIEW

File: utility/config.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this config
```
Comment (mohamad-ahmmad) [inline utility/config.txt:1]: Bad configuration

File: utility/utility.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+this is utility
```
Comment (mohamad-ahmmad) [inline utility/utility.txt]: This is not !!

File: utility/features.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+this is features imp
```
Comment (mohamad-ahmmad) [inline utility/features.txt:1]: this is not !!!! write a real code

**Review by mohamad-ahmmad — CHANGES_REQUESTED** [review]
You stupid

File: utility/config.txt (hunk 1 above)
Comment (mohmamad) [reply utility/config.txt:1]: okay

**Review by mohamad-ahmmad — APPROVED** [review]

**Pull request merged**